examples/rust/raw_mesh/src/main.rs

```diff
@@ -100,12 +106,6 @@ fn log_node(rec: &RecordingStream, node: GltfNode) -> anyhow::Result<()> {
 
 // --- Init ---
 
-// Use MiMalloc as global allocator (because it is fast), wrapped in Rerun's allocation tracker
-// so that the rerun viewer can show how much memory it is using when calling `show`.
-#[global_allocator]
-static GLOBAL: AccountingAllocator<mimalloc::MiMalloc> =
-    AccountingAllocator::new(mimalloc::MiMalloc);
```

Comment:
You should remove the dependency on `mimalloc` then also (or run `cargo machete`)